### DISCUSSION TITLE
How add element in html body with brython?

### DISCUSSION THREAD

**Discussion by jojo8356** [brython-dev/brython#2242] [Q&A]
How add element in html body with brython?, it's for upgrade html

**Comment by lulunac27a** [comment]
Use [`browser.html`](https://brython.info/static_doc/3.12/en/browser.html.html) module to create HTML element with uppercase tag names.  
For example:
```python
from browser import document, html
paragraph = html.P("Paragraph content")
document.body <= paragraph
```
